slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py

```diff
@@ -194,7 +199,7 @@ def onClickModels(self):
                 self.annotationModelSelector.addItem(model_name)
 
         self.segmentationButton.enabled = self.segmentationModelSelector.count > 0
-        self.updateGUIFromMRML()
+        self.onClickEditPoints()
```

Comment:
what is the logic behind this change?

Reply:
whenever user selects a different segment in the listbox.. clear previous and load the stored extreme points for that segment..